sql/core/src/main/scala/org/apache/spark/sql/Column.scala

```diff
@@ -689,6 +689,23 @@ class Column(protected[sql] val expr: Expression) extends Logging {
       println(expr.prettyString)
     }
   }
+
+  def | (other: Any): Column = BitwiseOr(expr, lit(other).expr)
```

Comment:
let's remove |, &, ^, and ~. The reason is that in Python they actually mean the non-bitwise version (because in Python we cannot override ||, &&).